config.yml

```
@@ -1,16 +1,34 @@
 accounts:
   - name: alice
-    coins: ["20000token", "200000000stake"]
+    coins: ["200000000stake","10000000mand"]
```

Let's change the order of coins for relevant accounts, because the first coin which is encountered the chain will make it as default denom. And we want `mand` to be the default denom. Just to be on safe side.
```suggestion
    coins: ["10000000mand", "200000000stake"]
```